DEVOPS-2337: Support TLS/SSL redis clients (#5)
waveaccounting/pylock: README.rst

-
http://www.regexprn.com/2010/05/using-memcached-as-distributed-locking.html

Notes related to the 0.4 release
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Added a poetry lock file, but did not setup circleci builds
To build and publish a release:
poetry publish -r gemfury -u $GEMFURY_PUSH_TOKEN -p NOPASS --build 

> I would suggest that if you are testing that your build patch releases till it works in your app THEN publish a minor or major version

Reviewer: ```suggestion
For testing I would suggest that you build and publish patch releases till it works in your app THEN publish a minor or major version
```
